src/python_routing_v01/compute_nhd_routing_SingleSeg.py

```diff
@@ -86,15 +121,17 @@ def _handle_args():
         dest="supernetwork",
         default="Pocono_TEST1",
     )
+    parser.add_argument(
+        "--parallel",
+        help="Use the parallel computation engine (omit flag for serial computation)",
+        dest="parallel_compute",
+        action="store_true",
+    )
 
     return parser.parse_args()
 
 
-ENV_IS_CL = False
-if ENV_IS_CL:
-    root = "/content/wrf_hydro_nwm_public/trunk/NDHMS/dynamic_channel_routing/"
-elif not ENV_IS_CL:
-    root = os.path.dirname(os.path.dirname((os.path.abspath(""))))
+root = pathlib.Path("../..").resolve()
```

Comment:
Refer to https://github.com/NOAA-OWP/t-route/issues/78#issuecomment-657788810
Do `pathlib.Path(__file__).parent.parent.resolve()`

Reply:
fixed. (not sure if I did that or @jdhondia-noaa ...)